### DISCUSSION TITLE
断网后，仍挂载硬盘的AList会导致系统的文件资源管理器崩溃，进而死机，请求解决方案

### DISCUSSION THREAD

**Discussion by teleen** [AlistGo/alist#5409] [Q&A]
使用Alist，使用 windows on arm设备，在需要登录验证的网络上进行工作，如果登录验证过期，导致网络服务中断，此时若Alist仍挂载着网络硬盘，会导致文件资源管理器崩溃，进而影响整个系统的运行；会导致系统死机，所有的系统组件无法运行，只能强行关机

**Comment by fisher44123** [comment]
加1 有解决办法了吗
